spec/statement.dd

```diff
@@ -1103,9 +1103,17 @@ $(GNAME StatementNoCaseNoDefault):
     $(GLINK ScopeBlockStatement)
 )
 
-        $(P $(EXPRESSION) is evaluated. The result type T must be
-        of integral type or $(CODE char[]), $(CODE wchar[]) or $(CODE dchar[]).
-        The result is
+        $(P $(I Expression) is evaluated.
+        If the type of the $(I Expression) is an enum,
+        it is repeatedly converted
+        to its $(GLINK2 enum, EnumBaseType) until it isn't. Then,
+        if the type is of integral type,
```

Comment:
A type does not have a type. So "if the type is integral" would be an alternative.